dockers/docker-orchagent/ndppd.conf.j2

```diff
@@ -21,7 +21,7 @@
 {%              set _x = proxy_interfaces[intf].append(prefix) %}
 {%          endif %}
 {%      endfor -%}
-
+route-ttl 2147483647
```

Comment:
How will it impact proxy nd replies?

Reply:
This should not have any impact, since we use the `static` configuration for replying to NS messages and do not forward the message to any interfaces.